pulp_rpm/app/tasks/comps.py

```diff
@@ -26,9 +27,8 @@ def parse_comps_components(comps_file):
     comps = libcomps.Comps()
     # Read the file and pass the string along because comps.fromxml_f() will only take a
     # path-string that doesn't work on things like S3 storage
-    with comps_file.file.open("rb") as f:
-        data = f.read()
-        comps.fromxml_str(data.decode("utf-8"))
```

Comment:
Are we losing anything by not doing the explicit decode()? I don't think so?

Reply:
I don't either